OrbitGl/TopDownView.h

```diff
@@ -7,12 +7,12 @@
 
 #include <unordered_map>
 
-#include "SamplingProfiler.h"
+#include "CaptureData.h"
 #include "absl/container/node_hash_map.h"
 
 class [[nodiscard]] TopDownNode {
```

Comment:
I think this nodiscard confuses clang-format

Reply:
I removed them in the file.